Resources/Plists/Info.plist

```diff
@@ -85,7 +87,7 @@
 			<key>LSItemContentTypes</key>
 			<array>
 				<string>com.sequel-ace.sequel-ace.spbundle</string>
-				<string>com.sequel-ace.sequel-ace.spbundle</string>
+				<string>com.sequel-ace.sequel-ace.sabundle</string>
```

Comment:
Does the order matter? If so, should we `prefer` `sa` over `sp` and change order in this + in the array above?

Reply:
Don't know, I've changed them for you though.